doc/nrf/ug_radio_fem.rst

```diff
@@ -61,6 +61,36 @@ Before you add the devicetree node in your application, complete the following s
    The MPSL library provides API to configure FEM.
    See :ref:`nrfxlib:mpsl_lib` in the nrfxlib documentation for details.
 #. Enable support for MPSL implementation in |NCS| by setting the :kconfig:option:`CONFIG_MPSL` Kconfig option to ``y``.
+#. Enable support for the FEM subsystem in |NCS| by setting the :kconfig:option:`CONFIG_MPSL_FEM` Kconfig option to ``y``.
+#. Chose the used FEM implementation by selecting an appropriate Kconfig option.
+
+The following FEM implementations are supported:
+
+* The nRF21540 GPIO implementation, see :ref:`ug_radio_fem_nrf21540_gpio`.
+  To use it, set the :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_GPIO` Kconfig option to ``y``.
+* The nRF21540 GPIO SPI implementation, see :ref:`ug_radio_fem_nrf21540_spi_gpio`.
+  To use it, set the :kconfig:option:`CONFIG_MPSL_FEM_NRF21540_GPIO_SPI` Kconfig option to ``y``.
+* The nRF21540 2-pin simple GPIO implementation.
+  To use it, set the :kconfig:option:`CONFIG_MPSL_FEM_SIMPLE_GPIO` Kconfig option to ``y``.
+
+Setting the FEM output power
+----------------------------
+
+The FEM gain to use by the simple GPIO FEM implementation is provided by the ``tx_gain_db`` property in devicetree and must represent the real gain of the FEM.
```

Comment:
typo: devicetree
device tree

Reply:
The spec name is "devicetree", I'd leave it as is (https://www.devicetree.org/)